src/Sulu/Bundle/WebsiteBundle/Resources/views/Extension/seo.html.twig

```diff
@@ -65,7 +65,7 @@
             {%- if defaultLocale == locale -%}
                 <link rel="alternate" href="{{ sulu_content_path(url, null, locale) }}" hreflang="x-default"/>
             {%- endif -%}
-            <link rel="alternate" href="{{ sulu_content_path(url, null, locale) }}" hreflang="{{ locale }}"/>
+            <link rel="alternate" href="{{ sulu_content_path(url, null, locale) }}" hreflang="{{ locale|replace('_', '-') }}"/>
```

Comment:
The problem is that we need two different representations of the localization on this line, explained it in more detail in https://github.com/sulu/sulu/issues/3174#issuecomment-276892216